packages/next/src/lib/find-config.test.ts

```diff
@@ -0,0 +1,68 @@
+import { mkdtemp, writeFile } from 'node:fs/promises'
+import { join } from 'node:path'
+import { tmpdir } from 'node:os'
+import { findConfig } from './find-config'
+
+describe('findConfig()', () => {
+  const exampleConfig = {
+    basePath: '/docs',
+  }
+  const configCode = {
+    mjs: `
+      const config = ${JSON.stringify(exampleConfig)}
+
+      export default config;
+    `,
+    cjs: `
+      const config = ${JSON.stringify(exampleConfig)}
+
+      module.exports = config;
+    `,
+  }
+  type TestPatterns = {
+    pkgConfigTypes: ('module' | 'commonjs')[]
+    exts: ('js' | 'mjs' | 'cjs' | 'package.json')[]
+  }
+  const testPatterns: TestPatterns = {
+    pkgConfigTypes: ['module', 'commonjs'],
+    exts: ['js', 'mjs', 'cjs', 'package.json'],
+  }
+
+  for (const pkgConfigType of testPatterns.pkgConfigTypes) {
+    for (const ext of testPatterns.exts) {
+      it(`should not append message several times (type: "${pkgConfigType}", config: ${
```

Comment:
Maybe separate `package.json` into a new test suite instead mixing with extensions? Will look more clear

Reply:
Thanks, I've applied in [17f4f6f](https://github.com/vercel/next.js/pull/63109/commits/17f4f6f338b5e50639818bcb7ad83e8a7bfa418a). Can you check if my change suits your expectation?